### PR TITLE
fix: wording changes to Gitpod plugins

### PR DESCRIPTION
**Pull request by loujaybee**
Some changes to the descriptions between the plugin types. 

**Main changes:** 
1. Isolate emphasis (bolding / italics) to the key aspects/differences 👀 
2. Add some additional points of clarification 🙏 

| Before      | After |
| ----------- | ----------- |
|  image | image |

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1957"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1957"></a>

### CODE REVIEW

File: generators/ide/jb-product.md [hunk 1]
```diff
@@ -49,17 +49,19 @@ Gitpod enables code editing using JetBrains {title} currently via [JetBrains Gat
...
-- [**JetBrains Gateway**](/docs/ides-and-editors/jetbrains-gateway#getting-started) running on _your local machine_ and provisioning **JetBrains Client** compatible with **{title} backend**. Plugins contribute new connectivity modes, like Gitpod Gateway plugin to connect to Gitpod workspaces.
+- [**{title} backend plugins**](#install-on-title-backend) - The JetBrains backend runs within the remote Gitpod workspace. Backend plugins contribute functionality requiring access to IDE experiences such as the remote filesystem (e.g. contributing support of languages and frameworks). When installed, **a backend plugin only applies to the currently running Gitpod workspace** and is not associated with a user. However, a plugin can be [preconfigured for all users of a repository](#preconfigure-for-repository) so that the plugin is enabled with every workspace start. It is not currently possible to install a backend plugin that applies to all workspaces of a Gitpod user or team.
```
Comment (akosyakov) [inline generators/ide/jb-product.md:56]: Backend plugins contribute the IDE experience requiring access to the remote backend such as  filesystem, tools, languages and frameworks.

Comment (akosyakov) [inline generators/ide/jb-product.md:56]: I don't there is the JetBrains backend, but many different like IntelliJ IDEA backend, or GoLand backend. I seen JetBrains docs either are using **IDE backend** for generalizazion. I think we are quite specific here and can use `{title} backend`

File: generators/ide/jb-product.md [hunk 2]
```diff
@@ -73,7 +75,11 @@ To install a plugin on **JetBrains Client** follow these steps:
 
 ### Install on {title} backend
 
-**{title} backend** running in a _remote Gitpod workspace_. Plugins contribute functionality which require access to remote filesystem and tools, e.g. contributing support of languages and frameworks. A plugin is installed only for [the current Gitpod workspace by default](#install-for-your-current-workspace), but can be [preconfigured for all users of a repository](#preconfigure-for-repository). It is not yet possible to install them for all workspaces of your user.
+The JetBrains backend runs within the remote Gitpod workspace. Backend plugins contribute functionality requiring access to IDE experiences such as the remote filesystem (e.g. contributing support of languages and frameworks).
```
Comment (akosyakov) [inline generators/ide/jb-product.md:78]: I think it should be the same like her https://github.com/gitpod-io/website/pull/1957/files#r859689767

`IDE experiences such as the remote filesystem` confuses me

In my head an algorithm is if a plugin needs access to something on remote backend then should install on backend, if it contributes to UI then on client.

**Review by akosyakov — APPROVED** [review]
@loujaybee I left couple comments to places which confused me.

Otherwise it is great improvement 🚀

File: generators/ide/jb-product.md (hunk 1 above)
Comment (loujaybee) [reply generators/ide/jb-product.md:56]: Not 100% sure of the change, @akosyakov, do you mean revert back from:

> **{title} backend plugins**

to 

> **{title} backend**

My thought process: It wasn't clear (to me, as a reader) if the block of text was describing the backend itself, or the process/considerations for backend plugins, more specifically. 

However, I'm not 100% sure if that's what you were referring to, though?

File: generators/ide/jb-product.md (hunk 2 above)
Comment (loujaybee) [reply generators/ide/jb-product.md:78]: Swapped:

> Plugins contribute functionality which require access to remote filesystem and tools, e.g. contributing support of languages and frameworks. 

For:

> Backend plugins contribute functionality for IDE experiences relating to the filesystem, tools or languages and frameworks

File: generators/ide/jb-product.md (hunk 1 above)
Comment (loujaybee) [reply generators/ide/jb-product.md]: Oh, you mean converting "JetBrains backend" to the specific "JetBrains IDE backend" / "JetBrains {title} backend"?

**Pull request merged**